buster/completers/base.py

```diff
@@ -35,28 +47,22 @@ def __init__(self, completion_kwargs: dict):
     def complete(self, prompt) -> str:
         ...
 
-    def generate_response(self, system_prompt, user_input) -> Completion:
+    def generate_response(self, system_prompt, user_input):
         # Call the API to generate a response
         prompt = self.prepare_prompt(system_prompt, user_input)
         logger.info(f"querying model with parameters: {self.completion_kwargs}...")
         logger.info(f"{system_prompt=}")
         logger.info(f"{user_input=}")
-        try:
-            completion = self.complete(prompt=prompt, **self.completion_kwargs)
-        except openai.error.InvalidRequestError:
-            logger.exception("Invalid request to OpenAI API. See traceback:")
-            return Completion("Something went wrong, try again soon!", True, "Invalid request made to openai.")
-        except Exception as e:
-            # log the error and return a generic response instead.
-            logger.exception("Error connecting to OpenAI API. See traceback:")
-            return Completion(
-                "Something went wrong, try again soon!", True, "Unexpected error at the generate_response level"
-            )
 
-        return Completion(completion)
+        completor = self.complete(prompt=prompt, **self.completion_kwargs)
+
+        self.completion = Completion(completor, self.error)
+
+        return self.completion
 
 
 class GPT3Completer(Completer):
+    # TODO: Update
```

Comment:
This is just gpt-3 right? turbo and 4 are in `ChatGPTCompleter`? If so, maybe we remove `GPT3Completer` for good?

Reply:
i added support for it, not sure we want to remove support, it is stil the only model that can be finetuned..